src/permitify/urls.py

```diff
@@ -19,9 +19,8 @@
 
 urlpatterns = [
     path('', views.index),
-    # admin - for rendering admin page
     path('admin', views.admin),
-    # process_request - for handling admin form submit
+    path ('process_request', views.process_request),
     path('submit_claim', views.submit_claim),
-    path('verify_dba', views.verify_dba)
+    path('verify_dba', views.verify_dba),
```

Comment:
The comma on this line looks like an error, if this is the last line in the array.  Can you check that please?